src/Tools/Metrics/MetricsOutputWriter.cs

```diff
@@ -61,7 +62,16 @@ void writeHeader()
                 switch (data.Symbol.Kind)
                 {
                     case SymbolKind.NamedType:
-                        writer.WriteAttributeString("Name", data.Symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat));
+                        var minimalTypeName = new StringBuilder(data.Symbol.ToDisplayString(SymbolDisplayFormat.MinimallyQualifiedFormat));
```

Comment:
I am not sure whether that's best to use a `StringBuilder` and `Insert` or to use a collection then `string.Join`, any suggestion?